Priyaa/HTML Basics Remake/Assignment-2.html

```diff
@@ -1,45 +1,58 @@
 <!DOCTYPE html>
 <html>
+
 <head>
```

Comment:
The metadata is missing. Use VS Code's autofill to make a basic HTML template (AKA Boilerplate).